src/egain/webChat/client/src/egainServiceDesk.ts

```diff
@@ -291,24 +286,13 @@ class EgainServiceDesk implements ServiceDesk {
   }
 
   /**
-   * Informs the service desk of a change in the state of the web chat that is relevant to the service desks. These
-   * values may change at any time.
+   * Informs the service desk of a change in the state of the web chat that is relevant to the service desks. These values may change at any time.
    *
    * @param state The current values of pieces of state from the main chat.
    */
   updateState(state: ServiceDeskStateFromWAC): void {
     this.userID = state.userID;
   }
-
```

Comment:
Oh dear, I seem to have lied to you. This function is declared as optional but it seems web chat is not treating it as optional and I'm getting errors when I type in the agent view (but they're just console logs, nothing is broken). I'm going to fix this in web chat. It's up to you but if you want to avoid these errors, you might want to put this code back. Even if I fix this in web chat, old versions will still generate these errors. Sorry about that :-(